ccd-definition-importer/templates/definition-importer.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "hmcts.ccddi.releaseName" . }}
+  labels:
+    {{- ( include "ccddi.labels" . ) | indent 4 }}
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-weight: "-5"
```

Comment:
Why is hook-weight changed? 
https://github.com/hmcts/chart-ccd/blob/3.x/ccd/templates/definition-importer.yaml#L13

Reply:
changed weight